e3nn/__init__.py

```diff
@@ -22,7 +56,18 @@ def set_optimization_defaults(**kwargs) -> None:
     for k, v in kwargs.items():
         if k not in _OPT_DEFAULTS:
             raise ValueError(f"Unknown optimization option: {k}")
-        _OPT_DEFAULTS[k] = v
+
+        # Handle the legacy jit_script_fx flag
+        if k == "jit_script_fx":
+            _OPT_DEFAULTS[k] = v
```

Comment:
Should this happen after the validation?

More generally, is the rationale for this to make sure that older code that looks at this flag can still find it in the defaults dict? Maybe worth documenting that with a comment if so